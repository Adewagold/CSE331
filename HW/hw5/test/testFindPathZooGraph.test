# Test BFS on zoo graph

# Parse through the data and build the graph
LoadGraph zoos zoos.tsv

# Find the shortest path Elephant to Tiger
FindPath zoos Elephant Tiger

# Find the shortest path Tiger to Elephant
FindPath zoos Tiger Elephant

# Find the shortest path Penguin to Monkey
FindPath zoos Penguin Monkey

# Find the shortest path Monkey to Monkey
FindPath zoos Monkey Monkey