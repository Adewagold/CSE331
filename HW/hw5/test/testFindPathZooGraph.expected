# Test BFS on zoo graph

# Parse through the data and build the graph
loaded graph zoos

# Find the shortest path Elephant to Tiger
path from Elephant to Tiger:
Elephant to Koala via Zoo3
Koala to Tiger via Zoo1

# Find the shortest path Tiger to Elephant
path from Tiger to Elephant:
Tiger to Koala via Zoo1
Koala to Elephant via Zoo3

# Find the shortest path Penguin to Monkey
path from Penguin to Monkey:
Penguin to Monkey via Zoo2

# Find the shortest path Monkey to Monkey
path from Monkey to Monkey: